exo/models.py

```diff
@@ -5,6 +5,8 @@
   "llama-3.1-8b": {
     "MLXDynamicShardInferenceEngine": Shard(model_id="mlx-community/Meta-Llama-3.1-8B-Instruct-4bit", start_layer=0, end_layer=0, n_layers=32),
     "TinygradDynamicShardInferenceEngine": Shard(model_id="mlabonne/Meta-Llama-3.1-8B-Instruct-abliterated", start_layer=0, end_layer=0, n_layers=32),
+    "LlamaCppDynamicShardEngine": Shard(model_id="lmstudio-community/Meta-Llama-3.1-8B-Instruct-GGUF", start_layer=0, end_layer=0, n_layers=32)
```

Comment:
Hey @AlexCheema, I'm having an issue here actually getting this model here to download. It is a valid huggingface model and I get no exception involving the download portion however, there is no progress being displayed. After some print debugging the ```download_repo_files``` just seems to exit right on this line https://github.com/exo-explore/exo/blob/87e08f89f1abd6711ee7686ac0bce02bbfb457d6/exo/download/hf/hf_helpers.py#L282

Again with no exception. Inferencing should be fairly straight forward after this is solved.